flashinfer/jit/kda_fp32_indexed_promotion.py

```
@@ -0,0 +1,1547 @@
+        "--source-root",
+        str(spec.artifact_root),
+        "--output-root",
+        str(output_root),
+        "--report",
+        str(report_path),
+        "--include-dir",
+        str(_cuda_include_dir()),
+        "--include-dir",
+        str(_get_include_dir()),
+    ]
+    result = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)